torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -247,7 +244,9 @@ def codegen_inputs(self):
                 self.prefix.splice(
                     f"""
                     if ((long({input_name}.data_ptr()) & ({GPU_ALIGN_BYTES} -1)) != 0) {{
-                        throw std::runtime_error("{input_name} is not aligned to {GPU_ALIGN_BYTES} bytes");
+                        AtenTensorHandle {input_name}_aligned;
```

Comment:
Should we warn here? Because misaligned inputs can significantly affect model performance

Reply:
Warning added